projects/packages/forms/src/blocks/contact-form/components/jetpack-field-consent.js

```diff
@@ -45,31 +58,54 @@ const JetpackFieldConsent = ( {
 				) }
 				insertBlocksAfter={ insertBlocksAfter }
 			/>
+			<JetpackFieldDimensionControls
+				clientId={ clientId }
+				setAttributes={ setAttributes }
+				width={ width }
+			/>
+			<InspectorControls group="styles">
+				<ToolsPanel
+					panelId={ clientId }
+					label={ __( 'Color', 'jetpack-forms' ) }
+					resetAll={ () => setAttributes( { labelColor: undefined } ) }
+					dropdownMenuProps={ toolsPanelDropdownMenuProps }
+				>
+					<div className="jetpack-field-controls__color-settings">
+						<ColorGradientSettingsDropdown
```

Comment:
This one needs the color data as well to show theme colors in the color palette.